src/Serilog.Sinks.XUnit/Serilog.Sinks.XUnit.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageTags>logging Serilog xUnit output ITestOutputHelper</PackageTags>
     <Copyright>Copyright © 2017 Todd Benning</Copyright>
     <Authors>Todd Benning</Authors>
-    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net9.0</TargetFrameworks>
```

Comment:
I wouldn't say that you need to remove support for older frameworks (and break other peoples projects), just bump the dependencies should be enough?